include/pfs/parsers.hpp

```diff
@@ -101,28 +106,36 @@ class file_parser
             std::string key;
             std::string value;
             std::tie(key, value) = utils::split_once(line, _delim);
-            if (key.empty())
+
+            std::string remapped_key = key;
```

Comment:
Maybe this part should come after the `utils::rtrim`... this way we don't have to do it twice.